src/drive/mobile/lib/cozy-helper.js

```diff
@@ -7,6 +7,8 @@ import { schema, DOCTYPE_FILES } from 'drive/lib/doctypes'
 export const getLang = () =>
   navigator && navigator.language ? navigator.language.slice(0, 2) : 'en'
 import { isMobileApp, getDeviceName } from 'cozy-device-helper'
+import appMetadata from 'drive/appMetadata'
```

Comment:
Hmm, we already have https://github.com/cozy/cozy-drive/blob/master/app.config.js#L51 

We should use only one, not both. https://github.com/cozy/cozy-drive/blob/6969df9c512053b0374db6c656a9bfbb5af85511/src/drive/mobile/lib/cozy-helper.js#L17

Reply:
ok, so the idea is to generate `appMetadata` in `app.config.js`?

Reply:
Ok, I use `appMetadata.version` everywhere.

I'm just not sure what to do about this:

```
  plugins: [
    new DefinePlugin({
      __PIWIK_SITEID__: 8,
      __PIWIK_TRACKER_URL__: JSON.stringify('https://matomo.cozycloud.cc'),
      __APP_VERSION__: JSON.stringify(pkg.version)
    })
  ]
```

Is `__APP_VERSION__` needed by piwik/matomo? I guess I can replace `pkg.version` by `appMetadata.version` here too.

Reply:
well, given that the file is `app.config.js` I wont do this change